openforcefield/utils/toolkits.py

```diff
@@ -4212,7 +4246,9 @@ def assign_partial_charges(
         # TODO: How should we implement find_executable?
         ANTECHAMBER_PATH = find_executable("antechamber")
         if ANTECHAMBER_PATH is None:
-            raise IOError("Antechamber not found, cannot run charge_mol()")
+            raise AntechamberNotFoundError(
+                "Antechamber not found, cannot run charge_mol()"
+            )
```

Comment:
Make sure to note the change in exception types in the `Behavior Changed` section of the release notes.